service.go

```diff
@@ -1749,8 +1753,15 @@ func (s *Service) GetPayloadTrusted(ctx context.Context, in *PayloadRequestParam
 		attribute.String("blockValue", payloadInfo.BlockValue),
 		attribute.String("uniqueKey", uKey),
 	)
-
 	span.SetAttributes(logMetric.GetAttributes()...)
+
+	// publish block to gateway
+	go func(tracer trace.Tracer, logger zerolog.Logger, payloadInfo *common.VersionedPayloadInfo, signedBeaconBlock *common.VersionedSignedBlindedBeaconBlock, client interface{}, authKey string) {
```

Comment:
wdyt of adding this to also getPayload (NotTrusted) especially if prefetch was successful.

Reply:
Can address in a different PR